CONTRIBUTING.md

```diff
@@ -86,16 +86,17 @@ The `cargo make run` command will start all the services and show a unified view
 
 ## Local Testing using Docker containers
 
-To spin up services using the latest code
+We recommend using linux VMs (if you are on MAC) to build the docker. 
 
-```bash
-# Clone golem
-cd golem
+### Example: 
 
-# Find more info below if you are having issues running this command(example: Running from MAC may fail)
-# Target has to be x86_64-unknown-linux-gnu or aarch64-unknown-linux-gnu-gcc
-cargo build --release --target x86_64-unknown-linux-gnu
+Use `multipass` or similar platforms, with a reasonable disk space (since it's the easiest to get a ubuntu)
```

Comment:
might be worth mentioning that we recommend ubuntu box with following dependencies installed on top of rust toolchain to compile golem:
```
sudo apt install pkg-config libssl-dev libfontconfig1-dev
```

As they might use their own tooling to get ubuntu vm (vagrant for example is really popular) or already have access to some other ubuntu box

Reply:
also dependent on you starting point, this might be needed
```
sudo apt install build-essential

```

Reply:
Also this 
```
sudo apt-get install protobuf-compiler
```